packages/api/src/controllers/taskController.js

```diff
@@ -638,6 +638,27 @@ const taskController = {
       return res.status(400).send(error);
     }
   },
+  async deleteTask(req, res) {
+    try {
+      const { task_id } = req.params;
+
+      const checkTaskStatus = await TaskModel.getTaskStatus(task_id);
+      if (checkTaskStatus.complete_date || checkTaskStatus.abandon_date) {
+        return res.status(400).send('Task has already been completed or abandoned');
+      }
+
+      // TODO: move to middleware
```

Comment:
I think this has been done and can be removed:
<img width="1287" alt="Screenshot 2023-04-09 at 10 45 07 AM" src="https://user-images.githubusercontent.com/30075973/230779564-4c48d11d-80c0-498a-886e-a314724592ff.png">

Reply:
Thank you!!